.github/workflows/build.yml

```diff
@@ -84,6 +85,7 @@ jobs:
           KNAPSACK_PRO_TEST_FILE_PATTERN: "{spec/controllers/**/*_spec.rb}"
         run: |
           git show --no-patch # the commit being tested (which is often a merge due to actions/checkout@v3)
+          bin/rails assets:precompile
           bin/rake knapsack_pro:rspec
```

Comment:
Can these be executed together in the one process? That might save some time.

``` suggestion
          bin/rails assets:precompile knapsack_pro:rspec
```

( i forget, but assume `rails` and `rake` can be used interchangeably)

Reply:
> Can these be executed together in the one process?

Good question. I will try that.

Reply:
It worked!